### ISSUE TITLE
X11 Support?

### ISSUE THREAD

**Issue by quintona** [liftoff/GateOne#417]
I know this is a strange question to ask in an open source project, but when do you plan on opening up the X11 support?

**Comment by davelab6** [comment]
http://www.phoronix.com/scan.php?page=news_item&px=MTUwNTk has a video of X11 support in GateOne...

**Comment by angelinux** [comment]
yes, but there is no source release. I am wondering if we can contribute to
the code, provided that is released as open source...

2015-04-08 2:05 GMT+02:00 Dave Crossland <notifications@github.com>:

> http://www.phoronix.com/scan.php?page=news_item&px=MTUwNTk has a video of
> X11 support in GateOne...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/liftoff/GateOne/issues/417#issuecomment-90764995>.
>



-- 
Where's the "Any" key?
Homer Simpson

**Comment by KR0SIV** [comment]
Does this code still exist? I'm not aware of anything that does X11 in a browser other than this. Would be nice to see a release and as mentioned, contribution as well.

**Comment by ocaisa** [comment]
X11 in a browser does exist, there are a few examples but I can't remember them all now, I was very impressed by Xpra and there's a nice article about how to use it through an ssh tunnel at https://hackaday.com/2017/03/31/linux-fu-applications-on-the-web/

**Comment by rohinrohin** [comment]
Yes, I would also like to know if there is x11 support from GateOne as per http://www.phoronix.com/scan.php?page=news_item&px=MTUwNTk. Do you guys know any other good x11 in a browser solutions?
